assets/prototype/domain/eventEditor/containers/queries/useDatetimeIds.js

```diff
@@ -0,0 +1,30 @@
+import * as R from 'ramda';
+import { useApolloClient } from '@apollo/react-hooks';
+import { GET_DATETIMES } from './dates';
+
+const useDatetimeIds = () => {
```

Comment:
@tn3rb you wrote that it should be named `useDatetimeIdsForTicket`, but as of what `useDeleteTicketMutation` is expecting this hook is not aware of any ticket information.

Reply:
ok great, that's even better then since there's less coupling